scripts/ci/lib.sh

```diff
@@ -694,7 +694,15 @@ image_prefetcher_system_await() {
 _image_prefetcher_prebuilt_await() {
     case "$CI_JOB_NAME" in
     *qa-e2e-tests)
+        if [[ "${JOB_NAME:-unknown}" =~ interop ]]; then
+            info "Image pre-fetch await for OCP-interop tests (disable exit)..."
+            enable -n exit
```

Comment:
TIL about `enable`!

However it would be great to see if we could **instead**:
1. bump the load balancer quota on the account which runs these tests (see my findings in https://issues.redhat.com/browse/ROX-27359). This would ensure that these jobs are not treated as second-class citizens w.r.t. image prefetching. Otherwise, if we don't have visibility into how they are performing, it would mean slow deterioration in the future.
2. if not possible, then just disable metric collection altogether for these jobs. There is no point launching the metric collection deployment/service and then waiting for it, if it's not going to succeed anyway. This would mean omitting [this flag](https://github.com/stackrox/stackrox/blob/bb9314cfebaa9be55f11ae791f28b135d2c4d8a0/scripts/ci/lib.sh#L648) at deployment time and returning from the await function before [this line](https://github.com/stackrox/stackrox/blob/bb9314cfebaa9be55f11ae791f28b135d2c4d8a0/scripts/ci/lib.sh#L774).

Reply:
I don't recall such variable... 🤔 
But completely disabling prefetcher will probably cause a lot of QA tests to fail, as a lot of deployments were changed to image pull policy `Never`.